server/routes/index.js

```diff
@@ -213,6 +213,31 @@ router.get('/api/v1/search/studies', ensureAuthenticated, function (req, res) {
   })
 })
 
+router.get('/api/v1/subjects', ensureAuthenticated, function (req, res) {
+  const { dataDb } = req.app.locals
+  dataDb
+    .collection('metadata')
+    .aggregate([
+      { $match: { study: { $in: JSON.parse(req.query.q) } } },
+      {
+        $addFields: {
+          numOfSubjects: { $size: { $ifNull: ['$subjects', []] } },
+        },
+      },
+      { $sort: { study: 1 } },
+    ])
+    .toArray(function (err, subjects) {
+      if (err) {
+        console.log(err)
+        return res.status(502).send([])
+      } else if (!subjects) {
+        return res.status(502).send([])
```

Comment:
Should this be `404`?

Reply:
I think these were just reformatted items, but we didn't work on this.